ddpui/api/airbyte_api.py

```
@@ -39,14 +60,15 @@
     TaskProgressStatus,
 )
 from ddpui.utils.singletaskprogress import SingleTaskProgress
+from typing import List
```

Ruff (0.12.2)

Redefinition of unused `List` from line 4

Remove definition: `List`

(F811)